### PR TITLE
Enable message internationalization with Transifex

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pytest==2.8.3
```
Comment (rphlo) [inline requirements-dev.txt]: add `-r requirements.txt` so to avoid having to include the default requirements

File: manage.py [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+        return True
+    return False
+
+
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kerrokantasi.settings")
-
+    if run_npm_script():
+        sys.exit(0)
     from django.core.management import execute_from_command_line
```
Comment (rphlo) [inline manage.py]: Is it that if there is some `scripts` in package.json we never run the django command?

Comment (akx) [inline manage.py]: No; `run_npm_script()` returns True iif it actually ran a command from package.json.

Comment (rphlo) [inline manage.py]: Ok I got it now... :)

File: requirements-dev.txt (hunk 1 above)
Comment (akx) [inline requirements-dev.txt]: Added.

File: manage.py (hunk 2 above)
Comment (akx) [inline manage.py]: Added a docstring, though. :)

**Pull request merged**